### PR TITLE
Points info

### PR DESCRIPTION
**Pull request by olore**
Display Average/Min/Max points

### CODE REVIEW

File: src/pages/game/players-table.jsx [hunk 1]
```diff
@@ -5,22 +5,83 @@ export function PlayersTable({ players }) {
...
+              {players.map(player => (
+                <tr key={player.name}>
+                  <td>{player.name}</td>
+                  <td>{player.points}</td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+        <div className="column">
```
Comment (BearAlliance) [inline src/pages/game/players-table.jsx]: I'd like to see this stats window as it's own component

**Comment by BearAlliance** [comment]
This would close #18

File: src/pages/game/players-table.jsx (hunk 1 above)
Comment (olore) [reply src/pages/game/players-table.jsx]: 💯I'm on it

**Comment by olore** [comment]
image

**Pull request merged**